format/private/format.sh

```diff
@@ -122,7 +119,22 @@ function ls-files {
           fi
           find_args+=("-name" "${patterns[$i]}")
         done
-        find "$@" "${find_args[@]}"
+        files=$(find "$@" "${find_args[@]}")
+    fi
+
+    if [[ $disable_git_attribute_checks == true ]]; then
```

Comment:
I imagine the use case for this flag is "I'm not using git" - would it be equivalent to just short-circuit here if there's no `.gitattributes` file or if `git` isn't on the PATH or `git` says "this folder isn't a repository"?

Reply:
Its a bit tricky since there can be one .gitattributes file per directory in a source repository so checking the root isn't sufficient and there's also the no arguments case where there's currently no way to avoid the call to `.git`. If we want to support gracefully handling git not being installed we should do it in its own PR. I did think about it though...

Reply:
that's okay, we don't even need to document this flag, it's just a break-glass. SGTM